### DISCUSSION TITLE
How to install it on my dev org?

### DISCUSSION THREAD

**Discussion by smelgin-absa** [codefriar/ApexKit#195] [General]
Hi @codefriar Kevin, I've just want to create a PoC about Feature Flags and Logs. How can I install your library on my Dev Org? Thanks in advance.

**Comment by VEERABABUSIDDI** [comment]
@codefriar HI Kevin, I am looking to install your library on my Dev Org, for doing POC can you please share me steps on how to enbale ApexKit in a dev sandbox
